ontologies/MIRO_sample_image_of_artwork.json

```diff
@@ -0,0 +1,107 @@
+; Comment: next step I assume would be to flesh out necessary context for this, including something 
+; to map these more human-readable terms to the ones in the .ttl ontologies
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "watu9tbc",
+  "type": "Work",
+  "identifiers": [
+    {
+      "label": "V0006724",
+      "authority": "MIRO"
+    },
+    {
+      "label": "ICV No 6939",
+      "authority": "Videodisk"
+    },
+    {
+      "label": ".b11579328",
```

Comment:
I'm wondering if the contents will actually be ".bnumber". The dot before the b indicates that it's from the system number index. The id in Sierra is bnumber, not .bnumber, so "b11579328", not ".b11579328".